dbms/src/Common/MyDuration.cpp

```diff
@@ -67,4 +67,10 @@ String MyDuration::toString() const
     auto frac_str = fmt::format("{:06}", microsecond);
     return fmt::format(fmt_str, sign > 0 ? "" : "-", hour, minute, second, frac_str);
 }
-} // namespace DB
\ No newline at end of file
+
+UInt64 calcSeconds(Int32 hour, Int32 minute, Int32 second)
```

Comment:
This function is pretty `TimeToSec`-dedicated. Maybe we could move it to `FunctionsDuration.cpp` to minimize its scope. Or even discard it and write this single line of code inside `TiDBTimeToSecTransformerImpl::apply()` == manually inline it.

Reply:
ok